index.js

```diff
@@ -54,8 +53,15 @@ function Generator (fn, opts) {
 
 	let time = 0, count = 0;
 
-	//return sync map
-	return function (buffer) {
+	//wrapper for convenience, there is nothing to end
+	generate.end = function () {};
+
+	return generate;
+
+	//return sync source/map
+	function generate (buffer) {
```

Comment:
Yeah, you can even declare variables with `var` after `return`, but since ES6 vars are no more in favor)